components/eamxx/src/control/atmosphere_driver.cpp

```diff
@@ -1031,19 +1031,19 @@ void AtmosphereDriver::set_initial_conditions ()
       auto& this_grid_topo_eamxx_fnames = topography_eamxx_fields_names[grid_name];
 
       if (fname == "phis") {
-        // The eamxx field "phis" corresponds to the name
-        // "PHIS_d" on the GLL and Point grids and "PHIS"
-        // on the PG2 grid in the topography file.
+        // For GLL points, phis corresponds to "PHIS_d" in the
+        // topography file. On PG2 grid, dynamics will take care
+        // of computing phis, so do not add to initialized fields.
         if (grid_name == "Physics PG2") {
-          this_grid_topo_file_fnames.push_back("PHIS");
+          // Skip
```

Comment:
This is a minor "fix". We don't need to load in `phis` on the PG2 grid, since dynamics internal dynamics->pg2 remapper will always overwrite this. Not a bug since nothing was going wrong, just unnecessary.